LocalSettings.php

```diff
@@ -5920,6 +5920,7 @@
 		'wisdomwikiwiki' => "//$wmgUploadHostname/wisdomwikiwiki/0/02/WWlogo.png",
 		'wisdomsandboxwiki' => "//$wmgUploadHostname/wisdomsandboxwiki/b/be/Sandbox_Logo.png",
 		'wishwiki' => "//$wmgUploadHostname/wishwiki/c/c6/Tamers_internet.png",
+		'kuonsamwiki' => "//$wmgUploadHostname/kuonsamwiki/b/bc/My_Handsome.png",
```

Comment:
this should be in alphabetical order

Reply:
Oh ok, should I change and request to commit again?

Reply:
You can just edit the current commit.

Reply:
@samkuonssp If you're unsure you could just make a request on https://phabricator.miraheze.org